lib/net.c

```diff
@@ -54,6 +52,23 @@ uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN] = {
 	0x6a, 0x42, 0xb7, 0x3b, 0xbe, 0xac, 0x01, 0xfa,
 };
 
+
+/* Function to randomize the default RSS key */
+void randomize_rss_key(){
+	srand(time(NULL));
+
+	/* key_initiator gets a random value each time which is xor'd with other values */
+	uint8_t key_intiator = rand();
+	default_rss_key[0] = default_rss_key[0]^key_intiator;
+
+	for(int i=1; i<GATEKEEPER_RSS_KEY_LEN; i++)
+	{
+		default_rss_key[i] = default_rss_key[i]^default_rss_key[i-1];
+	} 
+}
+
+randomize_rss_key();
```

Comment:
This line is a function prototype in C, not a function call. This means that you haven't even made sure that your pull request compiles. it's a big no-no. You should get a book about the C language and figure out where `randomize_rss_key()` must be called.